packages/components/molecules/f-media-element/src/FlexStyles.js

```diff
@@ -0,0 +1,117 @@
+import { MODIFIER_OPPOSITES_RULES_MAP, MODIFIER_RULES_MAP } from './constants';
+
+class FlexStyles {
+    flexDirection = '--row';
```

Comment:
Might be a stupid question (I don't use class syntax often).

Is there a reason these setup properties wouldn't be inside the constructor so that all of the initial values are grouped together?

Also, it makes more sense to me to have the constructor at the top of the class, rather than at the bottom as it is currently – just makes it a little clearer imo.

Reply:
Point 1 it's generally a rule that putting the class fields at the top allows you to document the object properties. I follow the [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes](url) for this. They quote the following "By declaring fields up-front, class definitions become more self-documenting, and the fields are always present."

Point 2 I agree constructor would be better suited at the top of the class. I will make that change.

Reply:
Makes sense. I would probably get rid of the extra line breaks between properties too

With the undefined properties, could they just be defined as

```
flexReverse;
operator;
…
```

Reply:
yea good point will make the change.

Reply:
Actually eslint requires the spacing?? but yes will make the change to remove undefined.

Reply:
Weird – what was the ESLint error out of interest? Can maybe look to update this in a separate PR then